www/include/configuration/configObject/host/host.php

```diff
@@ -116,6 +116,11 @@
         break;
     case "m":
         multipleHostInDB(isset($select) ? $select : array(), $dupNbr);
+        $acl = $centreon->user->access;
+        $aclDbName = $acl->getNameDBAcl('broker');
```

Comment:
this is used ?

Reply:
acl yes, the aclDbName maybe doesnt need to be updated